src/lib/replication/service/raft_repl_service.h

```diff
@@ -54,6 +54,7 @@ class RaftReplService : public GenericReplService,
 
 public:
     RaftReplService(cshared< ReplApplication >& repl_app);
+    ~RaftReplService();
```

Comment:
Since this is a derived class we should set this to `override` so we enforce the base classes have declared their destructors `virtual` yes?

Reply:
yes，we need declare a virtual destructor for the base class and set override in the derived clase.  thanks!